src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -51,16 +54,58 @@ public override void OnStopActivity(Activity activity, object payload)
                 return;
             }
 
+            string host;
+
+            if (context.Request.Host.Port is null or 80 or 443)
+            {
+                host = context.Request.Host.Host;
+            }
+            else
+            {
+                host = context.Request.Host.Host + ":" + context.Request.Host.Port;
+            }
+
+            TagList tags;
+
+#if NETCOREAPP
```

Comment:
We have a `net6.0` and `netstandard2.1` build of this instrumentation. Does this directive apply to those targets?

Reply:
Actually, now I'm wondering why we need a `netstandard2.1` target given that we have a `netcoreapp3.1` target.

Reply:
This directive only applies to `netcoreapp3.1` and `net6.0`. It does not apply to `netstandard2.1`\`netstandard2.0`

Reply:
could you add a comment why this compiler directive is relevant. ?

Reply:
@cijothomas - Added - Please review.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs#L243 -- could you check this part.. it does use reflection to get the releveant info on routes...

Reply:
We do not have that information available during OnStop event.